tests/api/controllers/gps_speed/gps_speed.c

```diff
@@ -54,16 +42,10 @@ int main(int argc, char **argv) {
                             speed);
 
   r = wb_gps_get_speed_vector(gps);
-  for (i = 0; i < 3; i++)
-    // clang-format off
-    // clang-format 11.0.0 is not compatible with previous versions with respect to nested conditional operators
-    ts_assert_double_in_delta(r[i], e[i], 0.000001,
-                              "The %c value measured by the GPS should be %g and not %g after %d wb_robot_step(s)",
-                              i == 0 ? 'X' :
-                              i == 1 ? 'Y' :
-                                       'Z',
-                              e[i], r[i], 2);
-  // clang-format on
+  ts_assert_vec3_equal(
+    r[0], r[1], r[2], NAN, NAN, NAN,
+    "The speed vector value measured by the GPS should be [%f, %f, %f] not [%f, %f, %f] after %d wb_robot_step(s)", e[0], e[1],
+    e[2], r[0], r[1], r[2], 2);
```

Comment:
I am not sure this is correct, you should have used `ts_assert_vec3_in_delta` against the `e` values instead of `NAN` here.

Reply:
C&P :)
Fixing it.
By the way, I realized that some of the ROS bit are in this project too.
In the RosGps.cpp/hpp if I am not mistaken.
Do you want me to update that too.

Reply:
> Do you want me to update that too.

Yes, please.

Reply:
Added. I will add the webots_ros part tomorrow or a bit later today.
In a branch of the develop branch I supposed ;) Same name.

Reply:
Damn forgot to update the doc for the ROS part.
adding that now.

Reply:
No idea. I guess this is fine this way.

Reply:
My colleague @BenjaminHug8 will review the ROS part.